common/Taskfile.yml

```
@@ -162,7 +164,7 @@ tasks:
       PATH:
         sh: git rev-parse --show-prefix
       REF:
-        sh: git rev-parse HEAD
+        sh: git rev-list -1 HEAD .
```

This makes it a little more explicit:

```suggestion
        sh: git rev-list -1 HEAD "{{ .USER_WORKING_DIR }}"
```